templates/vm/guest.xml.j2

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
Add the copyright and license indicator.
<!-- Copyright (C) 2023, RTE (http://www.rte-france.com) -->
<!-- SPDX-License-Identifier: Apache-2.0 -->